meta-swupd/recipes-core/swupd-server/swupd-server_2.53.bb

```diff
@@ -30,6 +33,6 @@ do_install_append () {
     install -m 0755 ${S}/test/signature/* ${D}${sysconfdir}/swupd-certs/
 }
 
-RDEPENDS_${PN} = "tar rsync"
+RDEPENDS_${PN} = "libarchive rsync"
```

Comment:
When replaced with bsdtar it breaks with

```
Parsing recipes: 100% |###############################################################################| Time: 00:00:09
Parsing of 1981 .bb files complete (0 cached, 1981 parsed). 2586 targets, 1279 skipped, 0 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies
ERROR: Nothing RPROVIDES 'bsdtar-native' (but virtual:native:/home/rojkov/work/ostro-os/meta-swupd/recipes-core/swupd-
server/swupd-server_2.53.bb RDEPENDS on or otherwise requires it)                                                    
NOTE: Runtime target 'bsdtar-native' is unbuildable, removing...
Missing or unbuildable dependency chain was: ['bsdtar-native']
ERROR: Required build target 'ostro-image-reference' has no buildable providers.
Missing or unbuildable dependency chain was: ['ostro-image-reference', 'swupd-server-native', 'bsdtar-native']
```